packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -869,19 +916,63 @@ export class LinuxBuildImage implements IBuildImage {
 
 /**
  * A CodeBuild image running Windows.
+ *
  * This class has a bunch of public constants that represent the most popular images.
  * If you need to use with an image that isn't in the named constants,
  * you can always instantiate it directly.
```

Comment:
This comment paragraph should be deleted (the constructor is now private).

Reply:
Fixed. We should extract these to a separate file in a subsequent commit (didn't want to mix up with this change)